test/generator/template_values_test.rb

```diff
@@ -8,36 +8,12 @@ def setup
       }
     end
 
-    def test_abbreviated_commit_hash
```

Comment:
Again, not core ruby functionality.
That it is implemented using an attr_reader is irrelevant.

Request: omit this commit.